models/session.go

```diff
@@ -0,0 +1,36 @@
+package models
+
+import (
+	"github.com/fatih/structs"
+	"time"
```

Comment:
Imports are not ordered 

It should be

Golang packages

3rd party packages

Repos own packages